Documentation/gettingstarted/k8s-install-gke.rst

```diff
@@ -34,14 +34,14 @@ Create a GKE Cluster
 
 You can apply any method to create a GKE cluster. The example given here is
 using the `Google Cloud SDK <https://cloud.google.com/sdk/>`_. This guide
-will create a cluster on zone ``europe-west4-a``; feel free to change the zone
-if you are in a different region of the globe.
+will create a single-zone cluster in ``europe-west4-a``; feel free to change
+the zone that is closer to where you are, you can create a regional cluster
```

Comment:
Nit:

```suggestion
the zone to one that is closer to where you are. You can create a regional cluster
```

Reply:
Thanks!

Reply:
Still looks like the original in the latest push (9910c88), maybe you forgot to amend?

Reply:
Yeah, thanks again!